packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -120,15 +120,15 @@ export const TabNonSsoItem: OptionItem = {
 
 export const M365SsoLaunchPageOptionItem: OptionItem = {
   id: "M365SsoLaunchPage",
-  label: "SSO enabled launch page",
+  label: `$(toolkit-tab) SSO enabled launch page`,
```

Comment:
Maybe we can localize the string as well?

Reply:
Yes, will do

Reply:
Updated